documentation/sequence diagrams/team/Create Team sequence.txt

```diff
@@ -0,0 +1,31 @@
+title Create Team
+participant User
+participant UI TeamList
+participant UI CreateTeam Dialog
+participant TeamView
+participant HttpRequest
+participant Team
+
+User->+UI TeamList: Click button Create
+UI TeamList->+UI CreateTeam Dialog: open CreateTeam Dialog
+User->UI CreateTeam Dialog: Enter data for creating a new team
+User->UI CreateTeam Dialog: Click button Submit
+alt not success
+UI CreateTeam Dialog->+TeamView: post(request: HttpRequest)
```

Comment:
post request before if/else statement
clarify success and not success statement